core/lib/brightcontent/base_controller_ext/pagination.rb

```diff
@@ -5,21 +5,59 @@ module BaseControllerExt
     module Pagination
       extend ActiveSupport::Concern
 
+      included do
+        helper_method :items_per_page
+        helper_method :current_page
+        helper_method :page_sizes
+        helper_method :corrected_page
+      end
+
       module ClassMethods
         def per_page_count
-          @per_page_count ||= 30
+          @page_size = page_sizes.min
         end
 
         def per_page(number)
-          @per_page_count = number
+          @page_sizes = [number]
+        end
+
+        def page_size_options(sizes)
+          return unless sizes.is_a? Array
```

Comment:
Would you like to make a technical debt issue for pointing brightcontent to rubygems again when the next version is released?